src/components/EventVolunteerList/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import ListItemAvatar from '@mui/material/ListItemAvatar';
+import Avatar from '@mui/material/Avatar';
+import { getRoleIcons } from '@/utils/role';
+
+// Object with name, role, and organization
+interface EventVolunteerResponse {
+  volunteer: {
+    name: string;
+    role: string;
+    organization: string;
+  };
+}
+
+// Prop Array with Objects
+interface EventVolunteerListProps {
+  volunteerResponses: EventVolunteerResponse[];
+}
+
+const EventVolunteerList: React.FC<EventVolunteerListProps> = ({
+  volunteerResponses,
+}) => {
+  return (
+    <List>
+      {volunteerResponses.map((eventVolunteer, index) => (
+        <ListItem key={index}>
+          {/* Display Icons of the Volunteer's Role */}
+          <ListItemAvatar>
+            <Avatar>{getRoleIcons([eventVolunteer.volunteer.role])}</Avatar>
+          </ListItemAvatar>
```

Comment:
The role isn't an avatar. The `getRoleIcons` returns the component you use. See `src/components/volunteers/Volunteer/index.tsx` for an example usage. If possible, I think it'd be good to have the icon underneath the volunteer name/organization